test/dif_pe_examples/vp/vp_general.json

```diff
@@ -38,7 +38,7 @@
           "EUDriversLicense"
         ],
         "issuer": "did:example:123",
-        "issuanceDate": "2010-01-01T19:73:24Z",
+        "issuanceDate": "2010-01-01T19:23:24.000Z",
```

Comment:
Why are we changing verifiable credentials invalidating the associated proofs?

Reply:
Would the claim be valid if the issuance date is invalid?

Reply:
Reverted!

Reply:
What would be invalid about the above data anyway. It is a valid UTC date, simply not including the miliseconds

Reply:
In the example above it's 19 hours, **73** minutes and 24 seconds. It fails to parse because 1 hour has 60 minutes and 73 is above of that.

Reply:
Didn't notice that. The spec for VC-data model doesn't take about the date format if I remember correctly. It does talk about it in the JWT (de)serialization though.

Indeed this value is odd, but that should surface in the verifier itself. These are simply the examples we are working with. As long as we don't break the proofs